examples/fluids/blasius.yaml

```diff
@@ -10,6 +10,11 @@ ts:
   #monitor_solution_interval: 10
 checkpoint_interval: 10
 
+ramp_amplitude: 1.0
+ramp_start: -3.1
+ramp_length: 0.2
```

Comment:
I think these need the new option names.

Reply:
Looks like you started the review before I caught this. I've actually just removed this from `blasius.yaml` all together, as I haven't tested it with the blasius anyways. That work with you?

Reply:
Yup!